packages/app/src/server/routes/apiv3/customize-setting.js

```diff
@@ -135,6 +136,10 @@ module.exports = (crowi) => {
     customizeScript: [
       body('customizeScript').isString(),
     ],
+    logo: [
+      body('attachmentId').isString().optional({ nullable: true }),
+      body('isDefaultLogo').isBoolean(),
+    ],
```

Comment:
`uploadedLogoSrc` seems to be missing in the validator

Reply:
Added `uploadedLogoSrc` to logo validator

Reply:
Change this to `customizedLogoSrc`